### PR TITLE
feat: new prometheus metrics added

### PR DESCRIPTION
**Pull request by taco-paco**

### CODE REVIEW

File: config-files/aggregator.yaml
Comment (Hyodar) [inline config-files/aggregator.yaml:15]: Shouldn't `enable_metrics` be here as well?

File: operator/attestor/attestor.go
Comment (Hyodar) [inline operator/attestor/attestor.go:56]:
```suggestion
type Attestorer interface {
	core.Metricable

	Start(ctx context.Context) error
```

Just seems a bit better to look at.

File: operator/attestor/event_listener.go
Comment (Hyodar) [inline operator/attestor/event_listener.go:33]: Currently we don't have more than one instance, but it would be better if all metrics registrations could check for existing metrics, as duplicate metrics directly panic

File: operator/consumer/event_listener.go
Comment (Hyodar) [inline operator/consumer/event_listener.go:33]: Same comment from `MakeAttestorMetrics` applies here (and in all other metrics registering)

**Review by Hyodar — APPROVED** [review]
Good, a few comments there, and after unit tests it's mergeable. As discussed today though, let's get it ready for merging but wait a little bit for updates on testing.

File: operator/attestor/attestor.go
Comment (taco-paco) [reply operator/attestor/attestor.go:56]: Done

File: operator/attestor/event_listener.go
Comment (taco-paco) [reply operator/attestor/event_listener.go]: Using non-panicking register now instead. Done

**Comment by Hyodar** [comment]
It would be good to add grafana to the setups too, I'll leave it to you to do it here or in another PR

**Comment by taco-paco** [comment]
> It would be good to add grafana to the setups too, I'll leave it to you to do it here or in another PR

Please send what technical tasks we have in DMs or open tickets in Notion. Don't feel comfortable with doing Grafana tables

**Comment by taco-paco** [comment]
Ping me once we can merge this PR

**Comment by Hyodar** [comment]
> Please send what technical tasks we have in DMs or open tickets in Notion. Don't feel comfortable with doing Grafana tables
Sure, we'd discussed about it yesterday. It's just adding the same grafana setup to `setup/operator` and `setup/aggregator`

File: config-files/aggregator.yaml
Comment (Hyodar) [reply config-files/aggregator.yaml:15]: ?

**Comment by taco-paco** [comment]
> > Please send what technical tasks we have in DMs or open tickets in Notion. Don't feel comfortable with doing Grafana tables
> 
> Sure, we'd discussed about it yesterday. It's just adding the same grafana setup to `setup/operator` and `setup/aggregator`

That part is in progress, will be added in another PR.

**Comment by taco-paco** [comment]
> Please check the comment above and resolve the conflicts, we can merge then

Ok, that is another concern. 

> but wait a little bit for updates on testing.

Was waiting for this. Will proceed with merging then

**Comment by Hyodar** [comment]
Feel free to merge this when the comment is solved and the conflict is resolved

File: config-files/aggregator.yaml
Comment (taco-paco) [reply config-files/aggregator.yaml:15]: done

**Pull request merged**
